setup.cfg

```diff
@@ -15,6 +15,9 @@
 # specific language governing permissions and limitations
 # under the License.
 
+
+
+
```

Comment:
Unrelated change?

Reply:
Sorry it might be accidentally changed. I'll revert this.